ext/nnstreamer/tensor_converter/tensor_converter_python3.cc

```diff
@@ -173,10 +173,15 @@ PYConverterCore::convert (GstBuffer *in_buf, GstTensorsConfig *config)
 
   if (output) {
     GstTensorInfo *_info;
-    unsigned int num_tensors = PyList_Size (output);
+    unsigned int num_tensors = (unsigned int) PyList_Size (output);
```

Comment:
Return of `PyList_Size`  coud be negative,  so how about checking error ?

```
unsigned int num_tensors = NNS_TENSOR_SIZE_LIMIT + 1;
Py_ssize_t out_len = PyList_Size (output);
if (out_len >= 0)
   num_tensors = (unsigned int) out_len;
```

Reply:
I changed param type (Py_ssize_t), please review the changes again.